pipelines/_ingredients/ApPipe.yaml

```diff
@@ -248,6 +248,7 @@ contracts:
   - detectAndMeasureDiaSource.doSkySources == standardizeDiaSource.doRemoveSkySources
   # Both loadDiaCatalogs and associateApdb connect to the APDB, so make sure they use the same configuration
   - loadDiaCatalogs.apdb_config_url == associateApdb.apdb_config_url
+  - calibrateImage.run_sattle == detectAndMeasureDiaSource.run_sattle
```

Comment:
Maybe a comment to the effect that `calibrateImage.run_sattle` isn't _strictly_ necessary but is an important optimization?